osf/models/institution.py

```diff
@@ -163,6 +167,56 @@ def save(self, *args, **kwargs):
         if saved_fields:
             self.update_search()
 
+    def _send_deactivation_email(self):
+        """Send notification emails to all users affiliated with the deactivated institution.
+        """
+        forgot_password = 'forgotpassword' if website_settings.DOMAIN.endswith('/') else '/forgotpassword'
+        attempts = 0
+        success = 0
+        for user in self.osfuser_set.all():
+            try:
+                attempts += 1
+                mails.send_mail(
+                    to_addr=user.username,
+                    mail=mails.INSTITUTION_DEACTIVATION,
+                    user=user,
+                    forgot_password_link='{}{}'.format(website_settings.DOMAIN, forgot_password),
+                    osf_support_email=website_settings.OSF_SUPPORT_EMAIL
+                )
+            except Exception:
+                logger.error('Failed to send institution deactivation email to '
+                             'user [{}] at [{}]'.format(user._id, self._id))
```

Comment:
nit: prefer keyword substitutions to format calls or use Python3 fstrings.
`'Failed to send institution deactivation email to user [{user}] at [{institution}]'.format(user=user._id, institution=self._id)`
or
`f'Failed to send institution deactivation email to user [{user._id}] at [{self._id}]'`